src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts

```diff
@@ -52,8 +53,11 @@ export class TableDesignerInput extends EditorInput {
 		return TableDesignerInput.ID;
 	}
 
-	get resource(): URI {
-		return undefined;
+	public get resource(): URI {
+		return URI.from({
+			scheme: Schemas.tmp,
```

Comment:
tmp? Where's that from?

Reply:
https://github.com/microsoft/azuredatastudio/blob/cb7b7da0a4bb2b45f0b6092e42fd1ca385938f09/src/vs/base/common/network.ts#L93

![image](https://user-images.githubusercontent.com/13777222/140584427-232e2125-8f27-41b0-9ecb-45f9037277c4.png)

looks to be the best fit for the table designer scenario.

Reply:
What about `vscodeCustomEditor`?

Reply:
Actually I'd be wary of using a schema here and having other side effects...for example in https://github.com/microsoft/azuredatastudio/blob/main/src/vs/workbench/contrib/customEditor/browser/customEditorInputFactory.ts#L126

It'd probably be safest just to make our own set of Schemas and just use that